spec/rubocop/options_spec.rb

```diff
@@ -189,7 +189,7 @@ def abs(path)
 
       it 'requires the passed path' do
         options.parse(['--require', required_file_path, 'example.rb'])
-        expect($stdout.string).to start_with('Hello from required file!')
+        expect($stdout.string).to a_string_starting_with('Hello from required file!')
```

Comment:
I agree with @bbatsov. As the [write-up](http://rspec.info/blog/2014/01/new-in-rspec-3-composable-matchers/#matcher-aliases) suggests, the `a_string_starting_with` method is an alias designed for cases where `start_with` is not grammatically correct, and where the generated failure message would be similarly inelegant.

By changing away from `start_with` here, we'd actually be _introducing_ the sort of hard-to-read grammar that, in different situations, the new aliases are designed to remove.